recipes/express/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "express" %}
+{% set version = "4.16.2" %}
+{% set sha256 = "e477b9f9507b0d7d1dec4129880f77ded4822eb8a9ae9f52de0aead01293bf3a" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/expressjs/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  noarch: generic
+  script: npm install -g .
+
+requirements:
```

Comment:
I think there are dependencies missing: https://github.com/expressjs/express/blob/master/package.json#L29

Note that it will be a little complicated to translate this into a list of conda dependencies because I do not think conda understands some of the version specifiers (e.g. `^1.2.1`)...

Reply:
@mwcraig: Thanks. I will add the dependencies to the list. But, will I have to push those deps on conda-forge if they are not?  During ci process, it looks for dep packages on conda-forge, and since "conda install -conda-forge packagename" doesn't work for the installation of js packages, I am not sure how it is going to work.

Reply:
@sannykr -- actually, I was completely wrong about the dependencies (didn't fully understand how node packages things).  Cross-reference to fuller discussion in one moment...

Reply:
On this thread: https://github.com/conda-forge/staged-recipes/pull/4473#issuecomment-346961665

At the (new) wiki page:  https://github.com/conda-forge/staged-recipes/wiki/Recipes-that-bundle-javascript-libraries

Reply:
Have had the same confusion before as well. Thanks for adding the wiki @mwcraig.